src/Files.App/Actions/FileSystem/CreateShortcutAction.cs

```diff
@@ -17,6 +17,7 @@ public RichGlyph Glyph
 			=> new(opacityStyle: "ColorIconShortcut");
 
 		public override bool IsExecutable =>
+			context.ShellPage is not null &&
```

Comment:
This action seems to work even if `context.ShellPage` is null, so I don't think this check is necessary.

Reply:
👍